lib/ansible/modules/database/influxdb/influxdb_user.py

```diff
@@ -98,6 +103,30 @@ def find_user(module, client, user_name):
     return name
 
 
+def check_user_password(module, user_name, user_password):
+    influxdb = InfluxDb(module)
+    client = influxdb.connect_to_influxdb()
+    try:
+        client.switch_user(user_name, user_password)
```

Comment:
[switch_user](https://github.com/influxdata/influxdb-python/blob/df70f5ce395d5650c14b9f68707220ab451b5ce8/influxdb/client.py#L192) function just set two properties, it doesn't call server:
```python
self._username = username
self._password = password
``` 
We should execute some command on server, that's why we call `get_list_users`.